src/modules/simulator/simulator.cpp

```diff
@@ -198,7 +153,6 @@ static void usage()
 {
 	PX4_WARN("Usage: simulator {start -[spt] [-u udp_port / -c tcp_port] |stop}");
 	PX4_WARN("Simulate raw sensors:     simulator start -s");
-	PX4_WARN("Publish sensors combined: simulator start -p");
```

Comment:
You need to remove the `-p` in the usage line. And probably the `-s` is not needed anymore either because it's the default, right?

Reply:
Removed `-p`, `-s` (default), and `-t` (removed previously).